doc/09-Automation.md

```diff
@@ -2,12 +2,12 @@
 
 It is possible to issue command actions without a browser. To do so, a form needs to be submitted by a tool such as
 curl. This is also used in the example below.
-The HTTP request method is `POST` if not documented otherwise and it might be necessary to manually configure this (e.g. `-X POST` for curl).
 
 ## Request Format
 
 The request is required to be an Icinga Web API request. For this it is necessary to transmit the `Accept` HTTP header
 and set it to `application/json`. In addition to this, the request must be authenticated using the `Basic` schema.
+The HTTP request method is `POST` if not documented otherwise and it might be necessary to manually configure this (e.g. `-X POST` for curl).
```

Comment:
The current content in this file does not exceed 120 characters per line, if avoidable. Please do the same.